app/presenters/step_nav_presenter.rb

```diff
@@ -7,7 +7,7 @@ def initialize(step_nav)
   def render_for_publishing_api(publish_intent = PublishIntent.minor_update)
     payload = required_fields
     payload.merge!(optional_fields)
-    payload.merge(publish_intent.present)
+    publish_intent.present.merge(payload)
```

Comment:
Other way round.  The hash given as the parameter overwrites the hash on which the method is called.  There's an example at https://apidock.com/ruby/Hash/merge.

I've swapped this since we want our options to overwrite the defaults.